CMakeLists.txt

```diff
@@ -4,10 +4,17 @@ project(mockturtle LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+# Options:
+#  - Release
+#  - RelWithDebInfo
+#  - MinSizeRel
+#  - Debug
+set(CMAKE_BUILD_TYPE Release)
+
 # Options
 option(MOCKTURTLE_EXAMPLES "Build examples" ON)
-option(MOCKTURTLE_TEST "Build tests" OFF)
-option(MOCKTURTLE_EXPERIMENTS "Build experiments" OFF)
+option(MOCKTURTLE_TEST "Build tests" ON)
+option(MOCKTURTLE_EXPERIMENTS "Build experiments" ON)
```

Comment:
The defaults should not be changed in the PR.

Reply:
Indeed, I should not have committed this file. I'll push a fix.